README.md

```diff
@@ -72,6 +72,42 @@ acme_php:
     resource: "@AcmePhpBundle/Resources/config/routing.xml"
 ```
 
+
+Usage
+-----
+
+Once the bundle is installed and configured, you can request your certificates
+by runnig the command `acmephp:generate`
+
+```bash
+$ ./bin/console acmephp:generate
+```
+
+The first time you run this command, AmePhpBundle will request a new
+certificate to the configured Certificate Autority (default
+[Letsencrypt](https://letsencrypt.org)) and store the generated certificate in
+the configured folder (default `~/.acmephp`).
+
+### Automatic renewal
+
+Each time you run the command `acmephp:generate` the certificate will be renew
+with a lifetime of 90 days (defined by the Certificate Autority). You can add
+a crontab to perform this task.
+
+```bash
+$ crontab -e
+
+0 0     1 * *     /var/www/my_app/bin/console acmephp:generate
+```
+
+> After regenerating a certificate you have to reload the web server to take
+the changes into account.
+
+> If you use a dedicated cron file in `/etc/cron.d/` be carrefull of the
+certificate storage location (configured by default in the `$HOME` directory)
+which is related to the user who run the command.
```

Comment:
It would be useful to be able to specify the storage directory as an option (like in the CLI).